js_modules/dagster-ui/packages/ui-core/src/instigation/TickDetailsDialog.tsx

```diff
@@ -92,6 +101,16 @@ const TickDetailsDialogImpl = ({tickId, instigationSelector}: InnerProps) => {
     );
   }
 
+  const firstTabLabel = () => {
+    if (tick.runIds.length || tick.originRunIds.length) {
+      return 'Requested runs';
+    }
+    if (tick.error) {
+      return 'Error message';
+    }
+    return 'Message';
```

Comment:
This was probably discussed earlier / in design, but I could also see calling this tab "Outcome" or "Result" instead of making it context-aware. It's a bit odd that when it says "Requested runs", there's another header beneath that says "Requested". (Maybe we just drop that second one?)

<img width="197" alt="image" src="https://github.com/dagster-io/dagster/assets/1037212/d5554980-af5a-4303-acc0-5f5ed5599777">